floris/simulation/turbine_multi_dim.py

```diff
@@ -0,0 +1,518 @@
+# Copyright 2023 NREL
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# See https://floris.readthedocs.io for documentation
+
+from __future__ import annotations
+
+import copy
+from collections.abc import Iterable
+
+import numpy as np
+import pandas as pd
+from attrs import define, field
+from flatten_dict import flatten
+from scipy.interpolate import interp1d
+
+# import floris.simulation.turbine as turbine
+from floris.simulation import (
+    _filter_convert,
```

Comment:
See the comment in the __init__ file. I suggest to change this to import the convert function directly from the module where it's stored.

Reply:
Updated the import statement.